content/kubermatic/master/tutorials_howtos/OPA_integration/_index.en.md

```diff
@@ -125,39 +120,213 @@ spec:
 ```
 
 - `constraintType` - must be equal to the name of an existing Constraint Template
-- `match` - works the same as [Gatekeeper Constraint matching](https://github.com/open-policy-agent/gatekeeper#constraints) 
-- `parameters` - holds the rawJSON parameters that are used in Constraints. As in Gatekeeper this can be basically anything, 
-Kubermatic just uses it as a raw json string to be able to structure it.
+- `match` - works the same as [Gatekeeper Constraint matching](https://github.com/open-policy-agent/gatekeeper#constraints)
+- `parameters` - holds the rawJSON parameters that are used in Constraints. As in Gatekeeper, this can be basically anything, Kubermatic just uses it as a raw JSON string to be able to structure it.
```

Comment:
Lets update this, rawJson should be deprecated so lets not comment on it anymore

```suggestion
- `parameters` - holds the parameters that are used in Constraints. As in Gatekeeper, this can be basically anything that fits the related Constraint Template.
```